src/pages/home/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { add, remove } from '../../redux/books/books';
+
+const Home = () => {
+  const books = useSelector((state) => state.counter.value);
+  const dispatch = useDispatch();
+  return (
+    <>
+      <div>{books}</div>
+      <button type="button" aria-label="add" onClick={() => dispatch(add())}>
+        Add book
+      </button>
+      <button type="button" aria-label="add" onClick={() => dispatch(remove())}>
+        Remove books
+      </button>
+    </>
```

Comment:
- [ ] This should be not Home but instead a BookList component placed in the components folder. It should also render a list of books using the `Book` component.

Reply:
Done! added book container component